src/main/java/com/revature/domain/QuestionPool.java

```diff
@@ -0,0 +1,174 @@
+package com.revature.domain;
+
+import java.io.Serializable;
+import java.sql.Date;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="ie_question_pool")
+public class QuestionPool implements Serializable {
+
+	@Id
+	@Column(name="qp_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="questionPoolSeq")
+	@SequenceGenerator(allocationSize=1, name="questionPoolSeq", sequenceName="question_pool_seq")
+	private Integer id;
+	
+	@Column(name="qp_question_text")
+	private String questionText;
+	
+	@Column(name="qp_max_communication_score")
+	private Integer maxCommunicationScore;
+	
+	@Column(name="qp_max_knowledge_score")
+	private Integer maxKnowledgeScore;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name="qp_sid")
+	private Subject subject;
+	
+	@Column(name="qp_count")
+	private Integer useCount;
+	
+	@Column(name="qp_last_date_used")
+	private Date dateLastUsed;
+	
+	public QuestionPool(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)